nss-day-cohort-48/rare-client-polite-pirates
# Jf view post details

#### Changes Made

1. Modified file `PostContent` to include `Title`, `Content`, `Category`, `Posted On`, and `Author`.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout jf-view-post-details
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
   

- [x] go to Posts
- [x] click on an article
- [x] check and see if all the content on the post is there
- [x] NOTE: this won't work if you haven't gone thru the refactoring time process from server side. How-to step list is on the Team Slack.

4. View code file.
   > Confirm file modifications are present as indicated above.
